plugins/modules/identity/ipa/ipa_dnsrecord.py

```diff
@@ -243,9 +263,13 @@ def ensure(module, client):
 
     ipa_dnsrecord = client.dnsrecord_find(zone_name, record_name)
 
+    record_values = module.params['record_values']
+    if 'record_value' in module.params:
```

Comment:
This will always evaluate to `True` since all parameters will always be in `module.params`. They will be `None` if they weren't specified by the user. So better do:
```suggestion
    if module.params['record_value'] is not None:
```